samples/bluetooth/mesh/light/child_image/hci_ipc.conf

```diff
@@ -1,6 +1,26 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
```

Comment:
```suggestion
# Copyright (c) 2024 Nordic Semiconductor ASA
```

Reply:
I wrote 2022 since it was back then this file originally was created. Somwhere in the file history the licence was removed, most likely by mistake.